package.json

```diff
@@ -3,15 +3,14 @@
   "description": "Translate module for Angular",
   "keywords": [
     "angular",
-    "angular2",
     "translate",
     "i18n"
   ],
   "main": "dist/index.js",
   "typings": "./dist/index.d.ts",
   "scripts": {
-    "test": "mocha --require core-js/shim --require reflect-metadata --require ts-node/register lib/**/*.spec.ts",
-    "prepublish": "npm run build",
+    "test": "mocha --require reflect-metadata --require ts-node/register lib/**/*.spec.ts",
+    "prepare": "npm run build",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
```

Comment:
Upgrading to latest semanding-release requires no pre/publish/post. Please modify semantic-release command to `semantic-release`